s3_target.txt

```diff
@@ -1 +1 @@
-vitessce-data/0.0.18/tile_ometiff
+vitessce-data/0.0.20
```

Comment:
Let's keep a subdirectory after the version number, unless there's a good reason to get rid of it? The version number can track the vitessce version number, but we don't want to forget that work for each branch within a version should, in general, also have its own space on S3.

Otherwise, fine.

Reply:
Oh, I didn't know that although I suppose that explains the structure.  I will update.